app/routes/_index.tsx

```diff
@@ -46,7 +45,7 @@ export default function Index() {
 						</h2>
 						{events.map((event, index) => (
 							<EventDetails
-								date={new Date(event.date)}
+								date={event.date}
```

Comment:
With Single Fetch these are de-serialized as `Date` instances so we don't need this anymore